src/main/kotlin/io/snyk/plugin/ui/jcef/OpenFileLoadHandlerGenerator.kt

```diff
@@ -26,25 +25,18 @@ class OpenFileLoadHandlerGenerator(
 
         val virtualFile = virtualFiles[filePath] ?: return JBCefJSQuery.Response("success")
 
-        ApplicationManager.getApplication().invokeLater {
-            val document = virtualFile.getDocument()
-            val startLineStartOffset = document?.getLineStartOffset(startLine) ?: 0
-            val startOffset = startLineStartOffset + (startCharacter)
-            val endLineStartOffset = document?.getLineStartOffset(endLine) ?: 0
-            val endOffset = endLineStartOffset + endCharacter - 1
-
-            navigateToSource(project, virtualFile, startOffset, endOffset)
-        }
+        val document = virtualFile.getDocument()
+        val startLineStartOffset = document?.getLineStartOffset(startLine) ?: 0
+        val startOffset = startLineStartOffset + (startCharacter)
+        val endLineStartOffset = document?.getLineStartOffset(endLine) ?: 0
+        val endOffset = endLineStartOffset + endCharacter - 1
 
+        navigateToSource(project, virtualFile, startOffset, endOffset)
         return JBCefJSQuery.Response("success")
```

Comment:
is this already running on the UI thread? or should you also use invokeLater here too?

Reply:
Ah never mind. Had to look in the implementation.